fileglancer_central/database.py

```diff
@@ -15,15 +15,15 @@ class FileSharePathDB(Base):
     """Database model for storing file share paths"""
     __tablename__ = 'file_share_paths'
     id = Column(Integer, primary_key=True, autoincrement=True)
-    name = Column(String, index=True, unique=True)
-    zone = Column(String)
-    group = Column(String)
-    storage = Column(String)
-    mount_path = Column(String)
-    mac_path = Column(String)
-    windows_path = Column(String)
-    linux_path = Column(String)
-    
+    name = Column(String(255), index=True, unique=True)
```

Comment:
Mysql needs the length - postgres works just like sqllite - it does not need them